dask/bag/tests/test_bag.py

```diff
@@ -190,6 +190,16 @@ def test_can_use_dict_to_make_concrete():
     assert isinstance(dict(b.frequencies()), dict)
 
 
+def test_from_url():
+    a = db.from_url(['http://google.com', 'http://github.com'])
+    assert a.npartitions == 2
+    a.compute()
```

Comment:
Should probably test something here.  Maybe connect up to the dask README on github and assert that `dask` is in the top five lines using take?